langgraph/src/pregel/write.ts

```diff
@@ -64,25 +66,45 @@ export class ChannelWrite<
     return input;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async _write(input: any, config: RunnableConfig): Promise<void> {
-    const values = this.channels.map(async ({ channel, value, skipNone }) => ({
-      channel,
-      value: await this.coerceValue(input, value, config),
-      skipNone,
-    }));
-
-    const valuesAwaited = await Promise.all(values);
-
+  async _getWriteValues(
```

Comment:
@nfcampos, the implementation of `_getWriteValues()` (and `_write()`) matches the Python implementation as specified in https://github.com/langchain-ai/langgraph/pull/345. However, I only made the bare minimum changes in `Pregel` and `StateGraph` to make the existing tests pass.

I'll add a new test to match this one: https://github.com/langchain-ai/langgraph/pull/345/files#diff-9f99beb90b66284c77ac257b4c0dfc06dcb998302f7c6740d84759460f704f53